modules/ROOT/pages/configuring-dev-env.adoc

```diff
@@ -72,6 +73,8 @@ target :'YourTarget' do
     ## Require one of the AeroGear SDK's here
     pod 'AGSPush', '{release-number}'
     pod 'AGSAuth', '{release-number}'
+    pod 'AGSCore', '{release-number}'
```

Comment:
Rather trivial but could we keep the same order of the imports in every platform?

Reply:
👍